docs/quickstart-guide/qs-overview.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+This guide will help you to quickly install and start working with Percona Everest.
```

Comment:
here "install" is used while at the bottom of the page" spin up"? What's the difference? Can we use one term if no difference?

Reply:
Updated